src-built-in/components/toolbar/src/fileMenu.jsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";

```

Comment:
Since this is just `jsx` file, this line could be just written as `import React from 'react'`

Reply:
Done.